includes/admin/class-wcmp-export.php

```diff
@@ -397,6 +397,13 @@ public function add_shipments(array $order_ids, bool $process, string $returnInT
             );
         }
 
+        $this->myParcelCollection = $collection;
+        $returnInTheBox           = WCMP()->setting_collection->getByName(WCMP_Settings::SETTING_RETURN_IN_THE_BOX);
+
+        if (WCMP_Settings_Data::NO_OPTIONS === $returnInTheBox || WCMP_Settings_Data::EQUAL_TO_SHIPMENT === $returnInTheBox) {
```

Comment:
Ik zou hier een helpertje van maken als het vaker voorkomt, en de check doen in `addReturnInTheBox` in plaats van hier.